packages/typescript-estree/README.md

```diff
@@ -215,7 +216,7 @@ interface ParseAndGenerateServicesOptions extends ParseOptions {
    * This means that if the parser encounters a file not included in any of the provided `project`s,
    * it will not error, but will instead parse the file and its dependencies in a new program.
    */
-  createDefaultProgram?: boolean;
+  deprecated__createDefaultProgram?: boolean;
```

Comment:
in regards to the comment, might be good to clean this up and make it more obtuse to help steer people away from it?

Like

```ts
/**
 * @deprecated - this flag will be removed in the next major.
 * Do not rely on the behaviour provided by this flag.
 */
```